docs/source/exceptions.rst

```diff
@@ -0,0 +1,6 @@
+Exceptions
+==========
+
+The following custom exceptions are defined dcos-e2e.
```

Comment:
I guess you mean "defined *in* dcos-e2e". Either way let's just do "defined." I think.